webapp/src/Controller/Jury/ExecutableController.php

```diff
@@ -96,14 +103,29 @@ public function indexAction(Request $request): Response
                 'link' => $this->generateUrl('jury_executable_download', ['execId' => $e->getExecid()])
             ];
 
-            $executables_table[]            = [
+            //dump((bool)count($e->getLanguages()));
+            dump((bool)count($e->getProblemsCompare()));
+            dump((bool)count($e->getProblemsRun()));
+
+            $executables_tables[$execType][] = [
+                'data' => $execdata,
+                'actions' => $execactions,
+                'link' => $this->generateUrl('jury_executable', ['execId' => $e->getExecid()]),
+                'cssclass' => count($e->getLanguages()) || count($e->getProblemsCompare()) || count($e->getProblemsRun()) ? '' : 'disabled',
+            ];
+            $executables_table[] = [
                 'data' => $execdata,
                 'actions' => $execactions,
                 'link' => $this->generateUrl('jury_executable', ['execId' => $e->getExecid()]),
+                'cssclass' => count($e->getLanguages()) || count($e->getProblemsCompare()) || count($e->getProblemsRun()) ? '' : 'disabled',
             ];
         }
+        //dump($executables_tables);
```

Comment:
more dumps

Reply:
We have a CI which also flags this, as this was WIP I forgot to remove them (but will when merging).